### DISCUSSION TITLE
ModuleNotFoundError for native node modules in getStaticProps after upgrade to 9.4

### DISCUSSION THREAD

**Discussion by wereHamster** [vercel/next.js#12856] [Help]
After upgrading from 9.3 to 9.4, I'm now getting a ModuleNotFoundError for 'fs' in getStaticProps.

Has anyone an idea what might cause it?

**Comment by Timer** [comment]
Could you please provide a reproducible demo so we can get this fixed?

**Reply by wereHamster** [reply]
I was hoping one of you could have an idea which change between 9.3 and 9.4 might cause it so I can narrow my search down. It's a huge private project and simplifying it will take a lot of time otherwise.

**Reply by wereHamster** [reply]
@Timer Is this sufficient as a reproducible demo? https://codesandbox.io/s/nextjs-94-getstaticprops-import-bug-6drb8
